README.md

```diff
@@ -157,17 +159,22 @@ In a template rendering a document:
 
 ```
 {% for field in doc.fields.get('$meta').schema.schema_fields %}
+  {# If the schema name is not set, skip. #}
+  {% if not field.schema_name %}
+    {% continue %}
+  {% endif %}
+
   {# Determine whether the field corresponds to a partial. #}
-  {% set basename = doc.collection.schemas_to_partials.get(field.schema_name) %}
+  {% set partial_filename = field.schema_name|kintaro_partial %}
```

Comment:
OK this is cool. :) I envision a few other functions here. Maybe we namespace it and make it more explicit, like the following?

`kintaro.schema_name_to_partial`

I also have an idea for one to fetch `ReferenceField` documents, e.g.

`kintaro.reference_to_doc` (returns the Grow `Document` corresponding to the reference)